src/devtools/client/webconsole/webconsole-ui.js

```diff
@@ -172,29 +135,22 @@ class WebConsoleUI {
     shortcuts.on(clearShortcut, event => this.clearOutput(true, event));
   }
 
-<<<<<<< HEAD
-  getLongString(grip) {
-    return this.webConsoleFront.getString(grip);
-  }
-
-=======
->>>>>>> 7818fe4e... Refactor console (part 2)
   /**
    * Sets the focus to JavaScript input field when the web console tab is
    * selected or when there is a split console present.
    * @private
    */
-  _onPanelSelected() {
+  _onPanelSelected = () => {
```

Comment:
I am curious about this style since it ends up putting a `;` at the end. Are you doing this only for event handlers to avoid the function binding?

Reply:
that's right. the two are identical, but it's nice to be specify it at the function level.